lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -81,6 +104,30 @@ defmodule MembraneLiveWeb.EventChannel do
     {:ok, Phoenix.Socket.assign(socket, %{peer_id: peer_id})}
   end
 
+  defp check_if_presenter(name, reloaded, id) do
+    [{_key, presenters}] = :ets.lookup(:presenters, id)
+    in_ets = if Enum.find(presenters, &(&1 == name)) != nil, do: true, else: false
+
+    case reloaded do
+      true ->
+        {:ok, in_ets}
+
+      false ->
+        if in_ets, do: remove_from_presenters(name, id)
+        {:ok, false}
+    end
```

Comment:
I think we can fully use case here, instead of switching if to case. 
```suggestions
    case {reloaded, in_ets} do
      {true, _in_ets} -> {:ok, in_ets}
      {false, true} ->
        remove_from_presenters(name, id)
        {:ok, false}
      {false, false} -> {:ok, false}
    end
```